pkgs/tools/misc/hyperfine/default.nix

```diff
@@ -19,6 +20,10 @@ rustPlatform.buildRustPackage rec {
 
   nativeBuildInputs = [ installShellFiles ];
   buildInputs = lib.optional stdenv.isDarwin Security;
+  propagatedBuildInputs = [
+    # hyperfine uses 'sh -c' which gets symlinked as alias for bash by this derivation
```

Comment:
Using propagatedBuildInputs is wrong here, would it be possible to patch the path in src